README.md

```diff
@@ -300,6 +280,62 @@ Default:
     "test-kitchen-instance" => <instance.name>,
     "test-kitchen-user"     => <env_user>,
 
+### Disk configuration
```

Comment:
I squeed a little that there were doc updates included. 💞 

![squeeing](https://user-images.githubusercontent.com/517302/39055331-f724e616-4481-11e8-96b1-1f41322f5e19.gif)